CondTools/RunInfo/interface/BTransitionAnalyzer.h

```diff
@@ -0,0 +1,72 @@
+#ifndef BTRANSITIONANALYZER_H
+#define BTRANSITIONANALYZER_H
+
+#include "FWCore/Framework/interface/one/EDAnalyzer.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/Run.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "FWCore/ServiceRegistry/interface/Service.h"
+#include "CondCore/CondDB/interface/Utils.h"
+#include "CondCore/DBOutputService/interface/PoolDBOutputService.h"
+#include "CondFormats/RunInfo/interface/RunInfo.h"
+#include "CondFormats/DataRecord/interface/RunSummaryRcd.h"
+
+namespace cond {
+  template<class T, class R>
+  class BTransitionAnalyzer: public edm::one::EDAnalyzer<edm::one::WatchRuns, edm::one::SharedResources> {
+  public:
+    BTransitionAnalyzer( const edm::ParameterSet& pset ):
+      m_currentThreshold( pset.getUntrackedParameter<double>( "currentThreshold", 18000. ) ) {
+      usesResource( "PoolDBOutputService" );
```

Comment:
@ghellwig you are absolutely right! This is common sense (as well as a software best practice):
* adding a static data member using constant expression for labelling the framework service name:`static constexpr auto serviceName = "PoolDBOutputService";`
* use it every time you refer to the service name in client code: `usesResource( std::string( PoolDBOutputService::serviceName ) );`